### PR TITLE
[new release] mdx (1.0.0)

### PR DESCRIPTION
**Pull request by samoht**
Executable markdown files

- Project page: <a href="https://github.com/realworldocaml/mdx">https://github.com/realworldocaml/mdx</a>
- Documentation: <a href="https://realworldocaml.github.io/mdx/">https://realworldocaml.github.io/mdx/</a>

##### CHANGES:

- Initial release

### CODE REVIEW

**Comment by mseri** [comment]
The CI is brokenish right now but this seems a proper failure

```
- File "lib/top/mdx_top.ml", line 416, characters 17-60:
- Error: This expression has type unit but an expression was expected of type
-          error:bool -> unit
```

File: packages/mdx/mdx.1.0.0/opam [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  "dune" {build}
+  "fmt"
+  "astring"
+  "logs"
+  "cmdliner"
+  "re" {>= "1.7.2"}
+  "ppx_tools"
+  "lwt" {test}
+]
+available: "ocaml-version" >= "4.06.0"
```
Comment (samoht) [inline packages/mdx/mdx.1.0.0/opam]: I am not sure this is a valid opam2 field. /cc @rjbou and @AltGr to check.

Comment (mseri) [reply packages/mdx/mdx.1.0.0/opam]: You are right. Opam2 requires a bound on ocaml in the deps field

**Comment by samoht** [comment]
I think I have fixed the constraints.

**Comment by mseri** [comment]
The CI agrees with you. A pity that this does not (yet) work with newer compilers

**Comment by samoht** [comment]
yes we need to use `omp`...

**Pull request merged**